### ISSUE TITLE
Suggestion: Store Focuser Step Size in NINA Profiles

### ISSUE THREAD

**Issue by BillRichards82** [ghilios/joko.nina.plugins#5]
The Hocus Focus plug-in is awesome, but I have several different setups resulting in 3 different focuser resolutions.  When I enter the Focuser Step Size into the Hocus Focus Aberation Inspector Options, that value is retained across NINA profiles. It would really be nice if the Focuser Step Size was stored in each NINA profile so the user doesn't have to remember to enter the correct value each time a different focuser configuration is used.

**Comment by ghilios** [comment]
Hi Domer82, I just checked the source code and step size is a per-profile setting. I may however need to force the settings to reload when there's a profile change. I'll look into that. Please try restarting NINA when changing profiles to confirm this for yourself.

**Comment by BillRichards82** [comment]
Next time I get a cahnce to set everything up again, I'll try that.  But when I tested it without shutting down NINA and restarting it, the value for the focuser last connected was retained for the new focuser.

**Comment by ghilios** [comment]
Domer, perhaps I misunderstood. Are you asking to store the settings per focuser instead of per profile?

**Comment by BillRichards82** [comment]
No, I think you understood correctly.  I have a bunch of different profiles (7+) that cover different telescopes, focal lengths, etc.  I have 3 different focusers that are used in those various profiles.  The focuser on my EdgeHD has a different step resolution when using the HyperStar because the secondary mirror is removed.  So the focuser resolution needs to be per profile.

The original request was to ensure that the focuser resolution was not only stored per profile, but loaded anew when a different profile is selected.  When I ran tests a few days ago, I loaded a profile, defined the focuser resolution (call it "X"), disconnected the equipment, connected the new equipment, loaded the corresponding profile, defined a different resolution (call it "Y"), then reverted back to the previous profile.  When I did that last step, the focuser resolution was still "Y", not "X" as it should be.

**Comment by ghilios** [comment]
Ahh gotcha. I can make this change, thanks for the report. I should do this across all of the my plugins

**Comment by BillRichards82** [comment]
Glad to be of help!

**Comment by BillRichards82** [comment]
Just FYI - After setting the focuser resolution in one specific profile last week and then exiting NINA, I launched NINA with the same profile last night and when I went into the Aberration inspector, that field was blank and I had to enter the value again.
